app/models/rdv.rb

```diff
@@ -1,11 +1,19 @@
 class Rdv < ApplicationRecord
+  include Webhook::Observable
+
   has_paper_trail
   belongs_to :organisation
   belongs_to :motif
   has_many :file_attentes, dependent: :destroy
   has_and_belongs_to_many :agents
   has_and_belongs_to_many :users, validate: false
 
+  def renderer
+    RdvBlueprint
```

Comment:
Je ne suis pas très satisfait de ça.

Reply:
a ruby hacker would use some magic to remove the need for this method ✨ 

in the concern you can do something like 

```
blueprint_class = "#{Rdv.new.class.name}Blueprint".constantize
blueprint_class.render(self, root: :data, meta: meta)
```

Reply:
:joy: